lib/Dialect/FIRRTL/FIRRTLFieldSource.cpp

```diff
@@ -57,6 +70,10 @@ void FieldSource::visitSubfield(SubfieldOp sf) {
 void FieldSource::visitSubindex(SubindexOp si) {
   auto value = si.getInput();
   const auto *node = nodeForValue(value);
+  if (!node) {
+    si.dump();
+    value.dump();
+  }
```

Comment:
```suggestion
```

Is this the same as #4646? Might be good to grab the spelling comments from that.  `it's`, `durring`.  Hyphen in "over-approximate" probably.

Reply:
apparently it is :)

Reply:
But now it has new stuff.